frame/staking/src/mock.rs

```diff
@@ -194,7 +199,7 @@ pub struct Test;
 
 parameter_types! {
 	pub const BlockHashCount: u64 = 250;
-	pub const MaximumBlockWeight: Weight = 1024;
+	pub const MaximumBlockWeight: Weight = frame_support::weights::constants::WEIGHT_PER_SECOND * 2;
```

Comment:
Having to switch this is a sign to me that potentially your solution is not very flexible. It is because you use the "block execution weight" right?

Reply:
This is because we use the `()` as weightInfo here and this produces real values, and even a small solution with 4 -5 nominators has a weight more than 1024. 

I think we should provide some `Null` implementation of `WeightInfo` for all pallets as well, for such cases.